### DISCUSSION TITLE
Question about MAT_CONST_H_MAX in paderborn.py

### DISCUSSION THREAD

**Discussion by moeKedama** [upb-lea/mag-net-hub#2] [General]
MAT_CONST_H_MAX = {
    "3C90": 84.7148502254261,
    "3C94": 64.8575649838852,
    "3E6": 74.1579701817075,
    "3F4": 150,
    "77": 86.5681744566843,
    "78": 87.5896894086919,
    "3C92": 150,
    "T37": 87.7490689795367,
    "3C95": 72.0625845199264,
    "79": 150,
    "ML95S": 150,
    "N27": 119.039616554254,
    "N30": 116.951204964406,
    "N49": 150,
    "N87": 100.674197407678,}  # in A/m    

I noticed that the MAT_CONST_H_MAX of some materials is set to 150, but their actual values may exceed this number. I don't quite understand the purpose of this operation, could someone provide an explanation, please? Thank you.

**Comment by wkirgsn** [comment]
Hi, these values denote the denominator during a simple scaling operation that represents the normalization of the data. 
NN training benefits from normalized data. 
Some materials are clipped to 150 A/m as we found during our analyses that the maximum occurring H values for these materials rather resemble outliers and would scale the entirety of the data inappropriately. 

Hope I could clarify these values ✌🏼
